README.md

```diff
@@ -29,15 +29,18 @@ demo_project:
   outputs:
     dev:
      type: spark
-     method: livy
+     method: cde
      schema: my_db
-     host: https://spark-livy-gateway.my.org.com/dbt-spark/cdp-proxy-api/livy_for_spark3/
+     auth_endpoint: https://service.spark-cde-gateway.my.org.com/
+     host: https://spark-cde-gateway.my.org.com/dex/api/v1/
      user: my_user
      password: my_pass
 ```
 
+- To obtain auth_endpoint follow the steps here: (https://docs.cloudera.com/data-engineering/cloud/api-access/topics/cde-api-get-access-token.html)
+
 ### Caveats
-- While using livy , in the Livy UI if you notice sessions change state to dead from starting instead of idle, make sure there is a proper mapping for the user in the IDBroker mapping section 
+- While using cde , in the Livy UI if you notice sessions change state to dead from starting instead of idle, make sure there is a proper mapping for the user in the IDBroker mapping section 
```

Comment:
Is this still relevant for cde

Reply:
We need to see this. I am unsure at the moment as we have tested this with user with IDBroker mapping.